tests/db_engine_specs/bigquery_tests.py

```diff
@@ -122,6 +122,18 @@ def test_extra_table_metadata(self):
         )
         self.assertEqual(result, expected_result)
 
+    def test_normalize_indexes(self):
+        """
+        DB Eng Specs (bigquery): Test extra table metadata
+        """
+        indexes = [{"name": "partition", "column_names": [None], "unique": False}]
+        normalized_idx = BigQueryEngineSpec.normalize_indexes(indexes)
+        self.assertEqual(normalized_idx, [])
+
+        indexes = [{"name": "partition", "column_names": ["dttm"], "unique": False}]
```

Comment:
we could add another small test with `indexes = [{"name": "partition", "column_names": ["dttm", None], "unique": False}]`